### PR TITLE
Create solution

### PR DESCRIPTION
**Pull request by al-kau**
generate uno-project (android, ios, maccatalyst, gtk, wasm, winui)
add .gitignore

### CODE REVIEW

**Review by ValeraFinebits — APPROVED** [review]

File: src/Eppie.App/Eppie.App.Mobile/Android/Resources/values/Strings.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="Hello">Hello World, Click Me!</string>
```
Comment (Gymbotan) [inline src/Eppie.App/Eppie.App.Mobile/Android/Resources/values/Strings.xml:3]: Do we need this string?

File: src/Eppie.App/Eppie.App.Mobile/Android/Resources/values/Strings.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    <string name="ApplicationName">UnoQuickStart</string>
```
Comment (Gymbotan) [inline src/Eppie.App/Eppie.App.Mobile/Android/Resources/values/Strings.xml:4]: Should we replace "Uno" to "Eppie"?

**Review by Gymbotan — APPROVED** [review]

**Review by anatolse — APPROVED** [review]

File: src/Eppie.App/Eppie.App.Mobile/Android/Resources/values/Strings.xml (hunk 1 above)
Comment (ValeraFinebits) [reply src/Eppie.App/Eppie.App.Mobile/Android/Resources/values/Strings.xml:3]: Will be fixed in the following commits.

File: src/Eppie.App/Eppie.App.Mobile/Android/Resources/values/Strings.xml (hunk 2 above)
Comment (ValeraFinebits) [reply src/Eppie.App/Eppie.App.Mobile/Android/Resources/values/Strings.xml:4]: Will be fixed in the following commits.

**Pull request merged**